src/shared/Microsoft.AzureRepos/AzureDevOpsRestApi.cs

```diff
@@ -72,22 +72,36 @@ public async Task<string> GetAuthorityAsync(Uri organizationUri)
                         if (tenantId != null)
                         {
                             _context.Trace.WriteLine($"Found {AzureDevOpsConstants.VssResourceTenantHeader} header with AAD tenant ID '{tenantId}'.");
-                            return authorityBase + tenantId;
+                            return new Uri(authorityBase, tenantId).ToString();
                         }
 
                         // If we have exactly one empty GUID then this is a MSA backed organization
                         if (tenantIds.Length == 1 && Guid.TryParse(tenantIds[0], out guid) && guid == Guid.Empty)
                         {
                             _context.Trace.WriteLine($"Found {AzureDevOpsConstants.VssResourceTenantHeader} header with MSA tenant ID (empty GUID).");
-                            return msaAuthority;
+                            return msaAuthority.ToString();
                         }
                     }
                 }
             }
 
             // Use the common authority if we can't determine a specific one
             _context.Trace.WriteLine($"Unable to determine AAD/MSA tenant - falling back to common authority");
-            return commonAuthority;
+            return commonAuthority.ToString();
+        }
+
+        private Uri GetAuthorityBaseUri()
+        {
+            // Check for developer override value
+            if (_context.Settings.TryGetSetting(
+                AzureDevOpsConstants.EnvironmentVariables.DevAadAuthorityBaseUri,
+                Constants.GitConfiguration.Credential.SectionName, AzureDevOpsConstants.GitConfiguration.Credential.DevAadAuthorityBaseUri,
+                out string redirectUriStr) && Uri.TryCreate(redirectUriStr, UriKind.Absolute, out Uri authorityBase))
```

Comment:
nit: tabbing for the method parameters (at this width, they should also be on their own lines), then also split the `&&` across multiple lines. This applies to the similar `if` statements in other parts of this PR.

Reply:
Ooh.. not sure what happened here in my style rules. Fixed.